webapp/tests/forms/steps/test_steuerlotse_step.py

```diff
@@ -468,20 +468,7 @@ class TestFormSteuerlotseStepCreateForm(unittest.TestCase):
     def attach_fixtures(self, test_request_context):
         self.req = test_request_context
 
-    def test_if_multiple_form_and_is_multiple_user_then_return_multiple_form(self):
-        self.req.form = MagicMock()
-        form_step = FormSteuerlotseStep(endpoint='lotse', header_title=None, stored_data={}, form=MagicMock())
-        form_multiple = MagicMock()
-        form_multiple_constructor = MagicMock(return_value=form_multiple)
-        form_step.form_multiple = form_multiple_constructor
-
-        with patch('app.forms.steps.steuerlotse_step.SteuerlotseStep.number_of_users', MagicMock(return_value=2)):
-            created_form = form_step.create_form(self.req, {})
-
-        form_multiple_constructor.assert_called_once()
-        self.assertEqual(form_multiple, created_form)
```

Comment:
I'm not sure we should delete that test. I think we still need a test to make sure the correct text is displayed. Maybe we can test, that babel is called with the correct num in the arguments (in my solution suggestion) or with the correct identifier with your solution?

Reply:
Yes, I agree with you, we could have a similar test. However, we would need to discuss where to put it because this is no longer shared behavior.